### PR TITLE
urls path added for bangazon_ui, include bangazon_ui.urls in the root…

### PR DESCRIPTION
**Pull request by juliakimchung**
# Description
added include url for bangazon_ui.urls in the root url
# Related Ticket(s)

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation

[ ] There is new documentation in this pull request that must be reviewed..

### CODE REVIEW

File: bangazon/bangazon_ui/templates/bangazon_ui/login.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {%extends 'bangazon_ui/index.html'%}
 {%block content%}
-<form class ="" action="{% url 'bangazon_ui:list' %}" method="post">
+<form class = "" action="{url bangazon_ui:list}" method="post">
```
Comment (ben-d-marks) [inline bangazon/bangazon_ui/templates/bangazon_ui/login.html]: This line needs `{%` and `%}` around actions

**Review by ben-d-marks — COMMENTED** [review]
Just change that one line

**Comment by shawn-mit** [comment]
please re-commit only file changed based on header for this PR and the branch name (jkc-login ) .

**Review by ben-d-marks — COMMENTED** [review]
Please keep PRs clean in the future: only files related to the task at hand. 
We should expect merge conflicts in `settings` and `urls`.
Otherwise, the files look good. 👍

**Comment by shawn-mit** [comment]
:rocket:

**Pull request merged**